sebsync.py

```diff
@@ -1,13 +1,18 @@
 """Synchronize Standard Ebooks catalog with local EPUB collection."""
 
 import click
+import hashlib
+import json
+import os
 import requests
+import time
 import xml.etree.ElementTree as ElementTree
 import zipfile
 
 from dataclasses import dataclass
 from datetime import datetime, timezone
 from pathlib import Path
+from platformdirs import *
```

Comment:
This will need to be added as a dependency in `pyproject.toml`.

Reply:
Added dependency via https://github.com/gfitzp/sebsync/commit/85aded69df223c8dc3ccdb7dccbd57850b9b1867#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R10